.github/workflows/testing.yml

```diff
@@ -27,11 +24,3 @@ jobs:
     - run: npm run test-coverage
       env:
         CI: true
-    - name: Upload coverage to Codecov
```

Comment:
i don't see the duplication part of this code (mentioned in the description). are we simply removing this?

Reply:
The npm command includes an upload by calling codecov. This Github action does the same thing. In that respect, it's a duplication of task.

Reply:
ah okay, got it